src/shogun/transfer/multitask/MultitaskKernelTreeNormalizer.h

```diff
@@ -143,14 +143,14 @@ class CNode: public CSGObject
  *	structure between tasks.
  *
  */
-class CTaxonomy : public CSGObject
+class CTaxonomy
 {
 
 public:
 
 	/** default constructor
 	 */
-	CTaxonomy() : CSGObject()
+	CTaxonomy()
```

Comment:
It is used by value-semantics, not pointers. Forces to use copy ctor which is not implemented with Self.